removed '.json' extension from service_schema.json

DaveVoyles opened the pull request:
If you pass in service_schema.json, the CLI throws an error:

```
{
    "Azure-cli-ml Version": "0.1.0a27.post3",
    "Error": "Error resolving dependency: no such file or directory service_schema.json"
}
```

*EX:* ```az ml manifest create --manifest-name mymanifest -f score.py -r python -s service_schema.json ```


However, if I run it *without* the .json:
```
az ml manifest create --manifest-name mymanifest -f score.py -r python -s service_schema     
```
RETURNS:
```
No model information provided, skipping model creation
Creating new driver at /var/folders/zg/wfn9qp9s5ys5qhjvxnk342k40000gn/T/tmpns10c73l.py
 score.py
 service_schema
Successfully created manifest
Id: 682eb200-f613-4d6e-8c01-b0991b68ecd3
More information: 'az ml manifest show -i 682eb200-f613-4d6e-8c01-b0991b68ecd3'
```

azurecla commented:
Hi __@snobu__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

ggailey777 commented:
@naziml can you verify this customer's tweak to the heading?

ggailey777 commented:
@tdykstra can you please verify for @naziml?

tdykstra commented:
@ggailey777 Looks good, corrects a typo

ggailey777 commented:
@snobu thank you for finding and fixing this typo. We appreciate your help improving the quality of Azure docs.

DaveVoyles committed:
removed '.json' extension from service_schema.json

If you pass in service_schema.json, the CLI throws an error:

```
{
    "Azure-cli-ml Version": "0.1.0a27.post3",
    "Error": "Error resolving dependency: no such file or directory service_schema.json"
}
```

*EX:* ```az ml manifest create --manifest-name mymanifest -f score.py -r python -s service_schema.json ```


However, if I run it *without* the .json:
```
az ml manifest create --manifest-name mymanifest -f score.py -r python -s service_schema     
```
RETURNS:
```
No model information provided, skipping model creation
Creating new driver at /var/folders/zg/wfn9qp9s5ys5qhjvxnk342k40000gn/T/tmpns10c73l.py
 score.py
 service_schema
Successfully created manifest
Id: 682eb200-f613-4d6e-8c01-b0991b68ecd3
More information: 'az ml manifest show -i 682eb200-f613-4d6e-8c01-b0991b68ecd3'
```
